### ISSUE TITLE
Change make_function``

### ISSUE THREAD

**Issue by neikeq** [godotengine/godot#12908]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**

**Issue description:**

**Steps to reproduce:**

**Link to minimal example project:**

**Comment by akien-mga** [comment]
The clock is ticking if you want to implement this for 3.0.

**Comment by reduz** [comment]
agreed, shuold be done, though i think it should also somehow work with visual script

**Comment by neikeq** [comment]
Please don't close this in the transition to [godotengine/godot-proposals](https://github.com/godotengine/godot-proposals) as I plan to implement it myself for 4.0. At least for text based scripts; I haven't thought about visual script yet.

**Comment by akien-mga** [comment]
Just commenting to clarify that this is still relevant despite the issue's age.

**Comment by YuriSizov** [comment]
This is still relevant, and there is even a note in code about this limitation. But changing it in 4.x as described would be a breaking change, so we would need to think of a better solution, perhaps.

**Comment by neikeq** [comment]
I'm no longer convinced about the solution I propose in my original message, because this feature still has problems with external editors, which is what most C# devs use.

Perhaps the code could be copied to the clipboard instead (displaying a toast notification).

**Comment by JensvandeWiel** [comment]
I think that that is a nice feature to have, even as extra.

**Comment by Calinou** [comment]
@JensvandeWiel Please don't bump issues without contributing significant new information. Use the :+1: reaction button on the first post instead.
